src/main/riddl/Tenant/tenantProjections.riddl

```diff
@@ -17,11 +17,11 @@ projector TenantViews is {
             and "set tenantName to TenantEstablished.info.name"
             and "set tenantInfo to TenantEstablished.info"
         }
-        on event TenantContext.InfoUpdated {
-            then "find tenant set to TenantEstablished.tenantId"
-            and "if name has changed, set tenantName to TenantEstablished.info.name"
-            and "set tenantInfo to TenantEstablished.info"
-        }
+        //on event TenantContext.InfoUpdated {
+        //    then "find tenant set to TenantEstablished.tenantId"
+        //    and "if name has changed, set tenantName to TenantEstablished.info.name"
+        //    and "set tenantInfo to TenantEstablished.info"
+        //}
```

Comment:
Why is this commented out?

Reply:
InfoUpdated was only used on Draft state. Really I should have asked if we want to be able to edit Tenant Info after creation, once we decided not to have Tenant Draft state. Too much going on, got left behind.

Reply:
Yes, Tenant Info needs to be editable once Established.  For example, the primary contact person changes or the credit card changes or ... . 

So, yes, let's not comment it out because TenantInfoUpdated event needs to still be handled

Reply:
This is still the command for Draft state. But I will add it to Established SuccessHandler